topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+All tools in Galaxy are kept up-to-date with their corresponding command-line tool. This is why in this tutorial we checked if the tool we want to use is up-to-date before using it. We then updated Megahit from 1.1.3.5 version to 1.2.9 (the latest version for that moment).
+
+The other case it might be necessary to update the tool when we get an error on tool run or we find out that there is a bug. It can be an issue/bug in the tool itself, then we have to contact the tool developers. In other cases, it can be an issue/bug with the tool wrapper. If this is the case, you may be able to fix it by yourself. 
+
+This was the case with the metaSPAdes tool. When we launched the metaSPAdes tool it failed with the error:
+
+ln: failed to create symbolic link 'paired_reads1/FASTQ_splitter_on_data_X.fastq': File exists
+
+The reason for this error was the same filenames with forward and reverse reads. We figure out that this error came from the FASTQ splitter tool, so we updated the FASTQ splitter. But we also updated spades tool to avoid similar cases in the future.
+
+Updating or fixing  tools has the same concept as adding tools but is less time consuming. Therefore, in this tutorial we will limit ourselves to fixing a tool, but the concept will be the same for adding or updating a new tool.
+
+Now we are going to update the tool in Galaxy by updating its wrapper. A wrapper is a XML which serves as a layer between the command-line tool itself and the Galaxy interface. This wrapper helps Galaxy users to have a visual interface and selection of the parameters of the command-line.  Please read our dedicated tutorial to learn more about Galaxy tools and the detailed instruction of how to build or update the tool is in {% cite noauthor_building_nodate %}.
+
+To update or fix a tool, we need to change its XML. Tools in a Galaxy server are installed by the administrators of this Galaxy server using the Galaxy ToolShed. The Galaxy ToolShed is like an AppStore with the XML for all possible Galaxy tools. So to update a tool, we need to change its XML in the ToolShed. 
+But usually, XML of tools are developed collaboratively inside GitHub repositories and then pushed to the ToolShed. So we need to find the development repository of the Galaxy tool to update it. 
+
+
+> ### {% icon hands_on %} Hands-on: Finding the development repository of a tool to update
+> 
+> 1. Open the fastqc splitter in Galaxy
+> 2. Click on “See in the ToolShed” on the top right of the tool form. You will be redirected to the tool on the ToolShed
+> 3. Follow the link in “**Development repository”**
+> 4. **You have now the tool wrapper, i.e. its XML**
+> 5. Once we have the development repository setup, we can update the tool. We recommend using Planemo, the Galaxy software development kit, to help the process. More information is in {% cite noauthor_building_nodate %}.
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Update the tools
```

Comment:
I don't think this is a hands-on, right? The tool is already updated, you're just documenting what was done, right? If so, please put it in a `details` box or so, rather than a hands-on

Reply:
I agreed